lib/matplotlib/patches.py

```diff
@@ -270,9 +269,11 @@ def _set_edgecolor(self, color):
 
     def set_edgecolor(self, color):
         """
-        Set the patch edge color
+        Set the patch edge color.
 
-        ACCEPTS: mpl color spec, None, 'none', or 'auto'
+        Parameters
+        ----------
+        color : color or None or 'auto'
```

Comment:
We have a :rc: alias, right?  Do we need a :color: alias so that all instances of color can point to the definition?

Reply:
Probably a good separate issue, per above.